libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -293,6 +297,9 @@ public void setAccountSwitcherActivityClass(Class<? extends AccountSwitcherActiv
      * @param name The name associated with the key.
      * @return The key used for encrypting salts and keys.
      */
+    /*
+     * TODO: Mark this deprecated and remove it in Mobile SDK 7.0.
+     */
     public String getKey(String name) {
```

Comment:
Should we mark the init methods that take a KeyImpl as deprecated and also remove them in 7.0?